trackhub/constants.py

```diff
@@ -1,749 +1,24 @@
 from __future__ import absolute_import
 
-from collections import OrderedDict
-from .validate import Parameter
-from . import validate
-
+from .parsed_params import param_defs, TRACKTYPES
 
 # http://genome-source.cse.ucsc.edu/gitweb/
 #       ?p=kent.git;a=blob;f=src/hg/makeDb/trackDb/README;hb=HEAD
 
+param_dict = {i.name: i for i in param_defs}
 
-track_field_order = OrderedDict((i.param, i) for i in [
```

Comment:
The following section was removed from the current delear/trackhub version and moved to `trackhub/parsed_params.py`. To merge this conflict, I've removed this section here and re-implemented our new parameters on the following conflicts.